plaidml/edsl/edsl.h

```diff
@@ -30,6 +30,7 @@ namespace details {
 struct Deleter {
   void operator()(plaidml_dim_expr* ptr) { ffi::call_void(plaidml_dim_expr_free, ptr); }
   void operator()(plaidml_expr* ptr) { ffi::call_void(plaidml_expr_free, ptr); }
+  void operator()(plaidml_exprs* ptr) { ffi::call_void(plaidml_exprs_free, ptr); }
```

Comment:
I will echo the Tensor/Tensors concern here for plaidml_expr/plaidml_exprs; I had to read this line a few times to see how it differed from the above. That said, I think we've been doing fine with plaidml_expr vs plaidml_exprs for a while so perhaps that's an argument for this being ok